build.gradle.kts

```diff
@@ -0,0 +1,156 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+  `java-platform`
+  `maven-publish`
+}
+
+defaultTasks("publishKnotxPublicationToMavenLocal")
```

Comment:
Is it required?

Reply:
I've changed it to `publishToMavenLocal` so it can be executed with simple `./gradlew`